utils/dataset_download.py

```diff
@@ -0,0 +1,57 @@
+import os
+from kaggle.api.kaggle_api_extended import KaggleApi
+from dotenv import load_dotenv
+
+def download_and_cleanup_kaggle_dataset():
```

Comment:
It is also better to return something, the dataset for example or the status of download